### PR TITLE
Add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by vhfmag**
First of all, great initiative!

This PR translates all i18n text files and exposes Brazilian Portuguese as a language choice in the platform. I haven't used the entirety of the platform (specially those features that required reputation), so I miss the context of some translated strings and mistakes have most certainly been made. I am a native Portuguese speaker and can speak English and French, so those languages' files served as basis for this translation, if it helps somehow.

Feedback is always appreciated :)

Changes:
- Translate into Portuguese the `app/i18n/ files`;
- Add Portuguese date in `app/i18n/i18n.js`;
- Add Portuguese in the default local in `app/components/App/LanguageSelector.jsx`.

### CODE REVIEW

File: app/i18n/i18n.js [hunk 1]
```diff
@@ -5,10 +5,12 @@ import dateLocaleFR from 'date-fns/locale/fr'
 import datelocaleEN from 'date-fns/locale/en'
 import datelocaleAR from 'date-fns/locale/ar'
 import datelocaleES from 'date-fns/locale/es'
+import datelocalePTBR from 'date-fns/locale/pt-BR'
```
Comment (Betree) [inline app/i18n/i18n.js:8]: It seems the `date-fns` version we're using doesn't support this, but latest definitely does: https://github.com/date-fns/date-fns/tree/master/src/locale.

We can either:
- Update `date-fns` to the latest version (if you have the time to do it)
- Or comment this line & merge with a `//TODO`, we'll be able to add it after the update

**Review by Betree — COMMENTED** [review]
Thank you for this contribution @vhfmag! We need to solve the issue with `date-fns`. The rest looks good to me (still need to test).

**Comment by vhfmag** [comment]
Sorry for submiting the PR without checking this!

I've updated date-fns to the latest version and handled the breaking changes. I must say I made some wild guesses when handling the fact that some old locale files ceased to exist (namely, `en` and `ar`) by replacing them with existing ones (`en-US`, `ar-MA`). I've tested the home page, which uses `TimeSince`; and video comments, which uses `CommentsReducer`. Those, together with `i18n.js`, should be the only files directly affected by the date-fns upgrade.

Something else noteworthy is that I had to add an explicit date and datetime format for arabic, since date-fns@2 requires a second argument for `format`. I don't know arabic at all, so I tried to stay safe by explicitly using date (`P`) and datetime (`Pp`) default formats, as per [date-fns i18n documentation](https://date-fns.org/v2.16.1/docs/format).

I hope I've covered everything! :)

File: app/i18n/i18n.js (hunk 1 above)
Comment (vhfmag) [reply app/i18n/i18n.js:8]: I've answered this comment outside of the thread, my bad!

File: app/components/Utils/TimeSince.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { connect } from 'react-redux'
-import differenceInSeconds from 'date-fns/difference_in_seconds'
-import distanceInWordsToNow from 'date-fns/distance_in_words_to_now'
+import differenceInSeconds from 'date-fns/differenceInSeconds'
+import distanceInWordsToNow from 'date-fns/formatDistance'
```
Comment (Betree) [inline app/components/Utils/TimeSince.jsx:4]: According to https://date-fns.org/v2.16.1/docs/formatDistanceToNow, there is a `formatDistanceToNow` helper that matches the current function signature. I think we should use that, with the current code when I post a new comment I see `In less than one minute`:
image

**Review by Betree — COMMENTED** [review]
It's looking great! Thanks for taking the time to migrate date-fns, that's super helpful.

One last comment below about the use of `formatDistanceToNow` and we should be good to go.

File: app/components/Utils/TimeSince.jsx (hunk 2 above)
Comment (vhfmag) [reply app/components/Utils/TimeSince.jsx]: Nice catch! It should be fixed by now :)

Comment (Betree) [reply app/components/Utils/TimeSince.jsx]: I still see this issue on https://deploy-preview-728--captainfact.netlify.app/, do you get a different result locally?

Comment (vhfmag) [reply app/components/Utils/TimeSince.jsx]: I can't seem to reproduce the issue on the preview deployment 😕 
The following screenshots are from [this video](https://deploy-preview-728--captainfact.netlify.app/videos/Jykl)

image
image
image
image

Comment (Betree) [reply app/components/Utils/TimeSince.jsx]: It only appears when you post a new comment. I'm currently testing on latest Firefox:

Peek 2021-01-20 19-31

Comment (Betree) [reply app/components/Utils/TimeSince.jsx]: My bad, seems to be resolved in the latest version :+1:

**Pull request merged**